.github/workflows/ui_tests.yml

```diff
@@ -95,6 +95,7 @@ jobs:
       - name: Run canary with minimal environment config
         env:
           NEXT_PUBLIC_PROJECT_ID: ${{ secrets.NEXT_PUBLIC_PROJECT_ID }}
+          MAILSAC_API_KEY: ${{ secrets.TESTS_MAILSEC_API_KEY }}
```

Comment:
The whole point of this minimal environment step is to have a minimum configuration. The canary doesn't need this API key and so it shouldn't be passed. This key isn't available when the canary runs.

Reply:
I see. Then I think we should remove the requirement of using `w3m-email-fixture.ts` which is importing `MAILSAC_API_KEY ` and causing tests to fail. 

I think this is a matter of the PR we introduced minimal test step. I'll take a look how we could do this.

Reply:
This regression happened before, see [this PR](https://github.com/WalletConnect/web3modal/pull/2106) which @tomiir helped refactor

Reply:
Is the canary now running new tests? Why was it working before and not anymore?

Reply:
@tomiir have no idea tbh. I think we have already open PR about it that I just saw: https://github.com/WalletConnect/web3modal/pull/2029

Reply:
The canary has been working, you can see it at the bottom of [this Grafana dashboard](https://g-e66b9a1a24.grafana-workspace.eu-central-1.amazonaws.com/d/prod-at-a-glance/prod-at-a-glance?orgId=1). Doesn't seem like #2029 affected it